src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.html

```diff
@@ -1,31 +1,81 @@
 <!-- TODO: Refactor Special Case 'Exercise' -->
 @if (sidebarItem) {
     <div class="mx-1 mb-1">
-        <div class="d-flex justify-content-between mb-1 align-items-baseline">
-            <span id="test-sidebar-card-title" class="small fw-semibold text-truncate me-2" [title]="sidebarItem.title">
-                @if (sidebarItem.icon) {
-                    <fa-icon [fixedWidth]="true" [icon]="sidebarItem.icon" />
-                }
-                {{ sidebarItem.title }}</span
-            >
-        </div>
-        <div class="d-flex justify-content-between align-items-baseline small">
-            <small class="me-2 text-truncate">
-                {{ sidebarItem.subtitleLeft }}
-            </small>
-            <small class="text-truncate d-inline">
-                @if (sidebarType === 'exercise' && sidebarItem.studentParticipation && sidebarItem.exercise) {
-                    <jhi-submission-result-status
-                        class="text-truncate result"
-                        [exercise]="sidebarItem.exercise"
-                        [studentParticipation]="sidebarItem.studentParticipation"
-                        [triggerLastGraded]="false"
-                        [isInSidebarCard]="true"
-                    />
-                } @else {
-                    {{ sidebarItem.subtitleRight }}
-                }
-            </small>
-        </div>
+        @if (sidebarType === 'exam') {
+            <div class="d-flex justify-content-between align-items-baseline">
+                <span id="test-sidebar-card-title" class="text-truncate me-2" [title]="sidebarItem.title">
+                    @if (sidebarItem.icon) {
+                        <fa-icon [fixedWidth]="true" [icon]="sidebarItem.icon" />
+                    }
+                    {{ sidebarItem.title }}
+                </span>
+            </div>
+            <div class="d-flex justify-content-between mb-1 align-items-baseline">
+                <div class="me-2 text-truncate">
+                    {{ sidebarItem.subtitleLeft }}
+                </div>
+            </div>
+            <div class="d-flex flex-column align-items-baseline">
+                <small class="me-2 text-truncate fw-semibold small-title-color">
+                    {{ 'artemisApp.courseOverview.sidebar.start' | artemisTranslate }}
```

Comment:
(optional) we prefer jhiTranslate over artemisTranslate

Reply:
I changed this in a follow-up PR too 😄

Reply:
But this file is going to be refactored by @rabeatwork anyway, so I think it would be better if I don't apply this change in this PR again